### ISSUE TITLE
{Content|View}Child[ren] with @deferred components

### ISSUE THREAD

**Issue by matanlurey** [dart-lang/angular#424]
_Not prioritized yet, just here for feedback._

```dart
// Method called when `deferredLib.loadLibrary()` is invoked.
void onLoad(Future<Null> loadLibrary) {
  // loadLibrary completes when deferred loading completes.
}
```~~

```html
<expensive-comp @deferred="onLoad"></expensive-comp>
```

## Update

We'll be supporting `@{Content|View}Child{ren}` instead of a new syntax.

See comment below.

**Comment by matanlurey** [comment]
Thinking out-loud, should `@ViewChild` support this instead?

```dart
@Component(
  selector: 'comp',
  directives: const [
    ExpensiveComp,
  ],
  template: r'''
    <expensive-comp @deferred></expensive-comp>
  ''',
)
class Comp {
  @ViewChild(ExpensiveComp)
  set comp(ExpensiveComp comp) {
    // Only called once deferred loading occurs.
  }
}
```

/cc @hterkelsen @leonsenft

**Comment by leonsenft** [comment]
That's actually quite elegant, I like it.

**Comment by hterkelsen** [comment]
LGTM also

**Comment by matanlurey** [comment]
So... it turns out this already works. We treat deferred content as an embedded view already.

I will add a test, and we can call it shipped!

**Comment by matanlurey** [comment]
Fixed in #1470.
